### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by numerogeek**
Add the YML version for the doctrine config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +138,17 @@ class Contact extends BaseContact
     </entity>
 </doctrine-mapping>
 ```
+YML Version : 
+```
+Acme\ContactBundle\Entity\Contact:
```
Comment (mremi) [inline README.md]: Can you add a comment to show the file path (like xml format). Thanks!

Comment (mremi) [inline README.md]: Ping @numerogeek

**Comment by numerogeek** [comment]
done @mremi !

**Comment by mremi** [comment]
Sorry, 3 more updates, can you:
1. add `yaml` after \`\`\` to add the syntax coloration
2. update the open tag (`#` instead of `<!--`) and remove the close one
3. squash your commits to keep only one

So much feedbacks just for a doc, sorry :)

Thank you!

**Comment by numerogeek** [comment]
@mremi is that better ? if it's not, close this PR I'll make a new one. I am not mastering git yet !

File: README.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ class Contact extends BaseContact
 ```
 
 ``` xml
-<!-- src/Acme/ContactBundle/Resources/config/doctrine/Contact.orm.xml -->
+<!--src/Acme/ContactBundle/Resources/config/doctrine/Contact.orm.xml -->
```
Comment (mremi) [inline README.md]: Can you revert it please? Better with the space :)

**Comment by mremi** [comment]
To squash the commits, use the interactive rebase of git. But as you want, if you prefer re-create a new PR, no problem :)

**Comment by mremi** [comment]
Fixed in #26

**Pull request closed without merging**